src/components/navbar/Navbar.js

```diff
@@ -23,12 +23,15 @@ import {
 import { HamburgerIcon, CloseIcon, ChevronDownIcon, ChevronRightIcon } from '@chakra-ui/icons';
 import { Link as LinkRouter } from 'react-router-dom';
 
+import LogoLight from '../icons/LogoLight';
 import LensBoosterIcon from '../icons/LensBoosterIcon';
 import Discord from '../icons/Discord';
 import Twitter from '../icons/Twitter';
 import Lens from '../icons/Lens';
 import Connect from './Connect.js';
 import SelectProfile from './SelectProfile.js';
+import { useEffect, useState } from 'react';
```

Comment:
Let's please put external imports (imports from other projects) on top and internal imports (imports from our project) below.